src/Components/ButtonElement/ButtonElement.tsx

```diff
@@ -10,7 +10,7 @@ export default function ButtonElement({
   type,
 }: ButtonElementProps) {
   return (
-    <button className={`${styles[type]} ${styles.buttonElement}`}>
+    <button type="submit" className={`${styles[type]} ${styles.buttonElement}`}>
```

Comment:
I do not expect that every `<ButtonElement>` is a submit button.
It feels confusing if I call it like `<ButtonElement type="active" />` , because `type` is usually `"button"` or `"submit"`.

What do you think about `<Button type="submit" active />` `<Button active={false} />`?

Anyway, what is the semantic behind `active` or `inactive`?
In CSS, `active` means, that the button is activated by the user (e.g. user presses down the primary mouse button):
https://developer.mozilla.org/en-US/docs/Web/CSS/:active

Maybe it makes sense to rename this to `variant` or `primary`?
`<Button type="submit" variant="primary" /> ` or `<Button type="submit" primary />`.

Examples of popular libs:
https://getbootstrap.com/docs/5.1/components/buttons/
https://mui.com/components/buttons/

In addition, the suffix `Element` is not required. Every React component is an Element.

Reply:
fixed